crates/fuel-core/src/service/sub_services.rs

```diff
@@ -102,10 +103,14 @@ pub fn init_sub_services(
 
     let last_height = *last_block_header.height();
 
+    let allow_historical_execution = config.combined_db_config.database_type
+        == DbType::RocksDb
+        && config.historical_execution;
```

Comment:
I think we need allow it on config level for the whole `fuel-core`, including the GraphQL and other services.

Reply:
bbbedd1247